common/Cargo.toml

```
@@ -22,15 +22,15 @@ bech32 = "0.9"
 bip39 = "1"
 cosmrs = "0.5"
 eyre = "0.6"
-ethers = { git = "https://github.com/gakonst/ethers-rs", rev = "0a031417d8ec9e8696229eb5d172f7155ec3c79c" }
+ethers = { version = "0.13", features = ["rustls"] }
```

we need features/fixes from the unreleased ethers... can it remain?
```suggestion
ethers = { git = "https://github.com/gakonst/ethers-rs", rev = "0a031417d8ec9e8696229eb5d172f7155ec3c79c", features = ["rustls"]  }
 ```

---

.cargo/git/checkouts/ethers-rs-c3a7c0a0ae0fe6be/0a03141/ethers-solc/src/resolver/parse.rs:52:57
    |
52  |                         SourceUnitPart::PragmaDirective(loc, _, pragma, value) => {
    |                                                         ^^^  ^  ^^^^^^  ^^^^^ expected 3 fields, found 4

---

There may be some issues with this version（0a031417d8ec9e8696229eb5d172f7155ec3c79c ） of rustls features

---

or maybe it can be updated to the latest commit (`2d8020eb3c0f34d998551362d796c2087a9439b7`): https://github.com/crypto-com/defi-wallet-core-rs/issues/407 ?

---

just note that you may need to bump `0a031417d8ec9e8696229eb5d172f7155ec3c79c` to `2d8020eb3c0f34d998551362d796c2087a9439b7` in all Cargo.toml files